src/bb-load.php

```diff
@@ -45,7 +45,7 @@ function handler_exception($e)
         print json_encode($result);
         return false;
     }
-
+}
```

Comment:
The closing curly brace must be on line 73 and not on line 48. 
The $page variable is the output when an exception is thrown and it is part of handler_exception function

Reply:
Thanks. I encountered an issue here and my system finally excepted the solution. It is now corrected as you suggested.